src/clients/QBittorrent.ts

```diff
@@ -285,16 +297,13 @@ export default class QBittorrent implements TorrentClient {
 		};
 	}
 
-	async isSubfolderContentLayout(searchee: Searchee): Promise<boolean> {
-		const response = await this.request(
-			"/torrents/files",
-			`hash=${searchee.infoHash}`,
-			X_WWW_FORM_URLENCODED,
-		);
-
-		const files: TorrentFiles[] = JSON.parse(response);
-		const [{ name }] = files;
-		return files.length === 1 && posix.dirname(name) !== ".";
+	async isSubfolderContentLayout(
+		searchee: Searchee,
+		torrentInfo: TorrentInfo,
+	): Promise<boolean> {
+		if (searchee.files.length > 1) return false;
+		if (dirname(searchee.files[0].path) !== ".") return false;
+		return dirname(torrentInfo.content_path) !== torrentInfo.save_path;
```

Comment:
same comment from the above about content path while files are in the "incomplete" directory or being moved FROM the incomplete directory?

Reply:
Let's continue this conversation on [this thread/review comment](https://github.com/cross-seed/cross-seed/pull/665#discussion_r1581031095)